project/src/store/film-data/film-data.ts

```diff
@@ -0,0 +1,48 @@
+import {Actions, ActionType} from '../../types/action';
+import {Film, FilmFromServerType} from '../../types/film';
+import {GenreList} from '../../types/genres';
+import {Review} from '../../types/reviews';
+import {FilmData} from '../../types/state';
+import {adaptToClient} from '../../utils/common';
+
+const initialState: FilmData = {
+  genre: GenreList.AllGenres,
+  films: [],
+  promoFilm: null,
+  currentFilm: {} as Film,
```

Comment:
```suggestion
  currentFilm: null,
```
Избегай такого. это очень плохая практика. Должно быть либо null (никакого фильма), либо НОРМАЛЬНЫЙ фильм